content/docs/policies/privacy/website.md

```diff
@@ -37,16 +37,17 @@ We might, however, use local storage capabilities to store basic preferences, su
 
 ## Analytics
 
-We use a self-hosted Plausible Analytics instance routed through our analytics subdomain to count website visits, downloads, etc. Plausible works without collecting any personal data or personally identifiable information, and without using cookies. You can see the same data we can see on the [public analytics dashboard](https://analytics.scratchaddons.com).
+We use a self-hosted Plausible Analytics instance routed through our analytics subdomain to count website visits, interactions, etc. Plausible works without collecting any personal data or personally identifiable information, and without using cookies. You can see the same data we can see on the [public analytics dashboard](https://analytics.scratchaddons.com).
 
 ## Feedback
 
 When sending feedback through <https://scratchaddons.com/feedback>, some data included in the request is:
 
 - Text written in the "identifiers" field, if any.
 - Text written in "feedback" field.
-- The version number for your Scratch Addons instance.
+- The version number of your Scratch Addons instance.
 - Your browser's user agent string (which often includes your browser's and operating system's names and version numbers).
+- Optionally, the list of currently enabled addons of your Scratch Addons instance.
 
 Feedback is sent to a public open source [Glitch.com Node.js server instance](https://glitch.com/edit/#!/scratchaddons-feedback) which then redirects the information to a [Discord](https://discord.com) server. We do not log the IP addresses of the users that send feedback. You can verify this by looking at the Glitch server source code linked above.

```

Comment:
Here is the other reference to Glitch. Is there a direct link to the source code on Napkin.io?

Reply:
@WorldLanguages

Reply:
When I said "next commit", what I meant is to bundle it with the changes for line 52, but oh well.

Reply:
> Is there a direct link to the source code on Napkin.io?

Yes: https://scratchaddons.npkn.net/feedback/

Reply:
```suggestion
Feedback is sent to a public open source [Napkin Node.js server instance](https://scratchaddons.npkn.net/feedback/) which then redirects the information to a [Discord](https://discord.com) server and an internal GitHub repository. We do not log the IP addresses of the users that send feedback. You can verify this by looking at the Glitch server source code linked above.
```